### PR TITLE
Added VisionMC 

### PR DESCRIPTION
**Pull request by MRSgami**
### General:

* `✓` The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* `✓` The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:

* `✓` My formatting matches the pre-existing formatting.
* `✓` There are no syntax errors.
* `✓` There is no pre-existing mapping with my id.
* `✓` My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - `✓` `id`: a lowercase string *(ex. `myserver`)*
  - `✓` `name`: a string *(ex. `MyServerPvP`)*
  - `✓` `addresses`: an array with lowercase strings *(ex. of address `my.server`)*
    - You do not need to specify sub-domains, Lunar Client services automatically detect them.

### Logo Additions or Updates:

* `✓` My image is a `.png` file.
* `✓` I uploaded the image in the `logos` directory.
* `✓` My image is transparent & square (1:1 aspect ratio).
* `✓` My image is a minimum of `512` pixels in width and height.
* `✓` My images image name matches its mapping id.

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 [
+     {
+        "id": "visionmc",
+        "name": "VisionMC Network",
+        "addresses": [
+            "visionmc.club"
+            "na.visionmc.club"
```
Comment (GrowlyX) [inline servers.json:7]: Please look over the checklist again.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

File: servers.json (hunk 1 above)
Comment (MRSgami) [reply servers.json:7]: This is what we wanted to be added the others were a mix up this one is correct.

Comment (MRSgami) [reply servers.json]: It seems fine I do not see any issue?

Comment (MRSgami) [reply servers.json]: All good

File: servers.json [hunk 2]
```diff
@@ -1,1673 +1,1680 @@
...
-    {
-	"id": "valleypvp",
-	"name": "ValleyPvP Network",
-	"addresses": [
-	    "valleypvp.us"
-  	]
-    }
-]
+[
+     {
```
Comment (GrowlyX) [inline servers.json:2]: Please remove this unneeded whitespace before the starting bracket.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

**Comment by MRSgami** [comment]
Updated Server Mappings List

**Comment by MRSgami** [comment]
Should be all good now

**Pull request closed without merging**